delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/ElasticSearchQuery.scala

```diff
@@ -27,15 +27,13 @@ trait ElasticSearchQuery {
     * @param project      the project where the view exists
     * @param query        the elasticsearch query to run
     * @param qp           the extra query parameters for the elasticsearch index
-    * @param sort         the sorting configuration
     */
   def query(
```

Comment:
It is alright that we are not supporting SortList on elasticsearch queries, but then we have to be consistent and do the same in `ElasticSearchViewsQuery` and `ElasticSerchRoutes`

Reply:
what do you think about the `qp` should I remove these as well?

Reply:
No. That's useful and part of the elasticsearch API